### PR TITLE
Change notification flow to not trigger an error on canceled payments

### PR DESCRIPTION
**Pull request by andercampanha**
Cypress clears all stubs between tests by default: https://docs.cypress.io/api/commands/stub.html#Restores

### CODE REVIEW

File: tests/magento/cypress/integration/brazil.js [hunk 1]
```diff
@@ -100,6 +100,20 @@ describe('Shopping', () => {
           });
         });
       });
+
+      it('will fail to `blue horizons bracelets` and notification won`t be error', () => {
```
Comment (miguelxpn) [inline tests/magento/cypress/integration/brazil.js]: Just some naming pedantism, but shouldn't it be 'will fail to buy'?

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
-# EBANX Payment Gateway for Magento 1.9.x [![Build Status](https://travis-ci.com/ebanx/magento-gateway-ebanx.svg?branch=master)](https://travis-ci.com/ebanx/magento-gateway-ebanx) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/64dbf4b4e1b941b7bb38f35251af7575)](https://www.codacy.com/app/ebanx/magento-gateway-ebanx)
+# EBANX Payment Gateway for Magento 1.9.x 
+[![Build Status](https://img.shields.io/travis/ebanx/magento-gateway-ebanx/master.svg?style=for-the-badge)](https://travis-ci.com/ebanx/magento-gateway-ebanx)
```
Comment (andercampanha) [inline README.md]: Codacy

File: app/code/community/Ebanx/Gateway/controllers/PaymentController.php [hunk 3]
```diff
@@ -68,25 +92,41 @@ public function notifyAction()
...
+     * @throws Ebanx_Gateway_Loadcanceledcardpaymentexception When CA card payment not found
      */
     private function initialize()
     {
         $this->helper = Mage::helper('ebanx/order');
         $this->validateEbanxPaymentRequest();
         $this->hash = $this->getRequest()->getParam('hash_codes');
         $this->loadOrder();
-        $this->statusEbanx = $this->loadEbanxPaymentStatus();
+        $this->statusEbanx = $this->loadEbanxPaymentInfo();
```
Comment (andercampanha) [inline app/code/community/Ebanx/Gateway/controllers/PaymentController.php]: Revert this change

File: app/code/community/Ebanx/Gateway/controllers/PaymentController.php [hunk 4]
```diff
@@ -265,4 +305,26 @@ private function validateStatus()
...
     }
+
+    /**
+     * @param string $paymentTypeCode
+     *
+     * @return bool
+     */
+    private function isCreditCardPayment($paymentTypeCode)
+    {
+        $creditCardBrands = array(
```
Comment (andercampanha) [inline app/code/community/Ebanx/Gateway/controllers/PaymentController.php]: Following lib-JS possible brands

Comment (frop) [reply app/code/community/Ebanx/Gateway/controllers/PaymentController.php]: :+1:
